src/resolvers/mutation/create-deployment/index.js

```diff
@@ -108,12 +108,29 @@ export default async function createDeployment(parent, args, ctx, info) {
     ...args.properties
   };
 
+  let serviceAccountAnnotations;
```

Comment:
```suggestion
  const serviceAccountAnnotations = {};
```

This way you can keep it a constant, but by making it an object, we can still append keys down below.

Reply:
fixed.